src/components/resourcesResultsPanel.js

```diff
@@ -1,80 +1,141 @@
 import React from "react"
 import Button from "./button"
+import { BLOCKS } from "@contentful/rich-text-types"
+import { documentToReactComponents } from "@contentful/rich-text-react-renderer"
 import { determineSlug } from "./helpers"
 import "./resourcesResultsPanel.scss"
 
 const ResourcesResultsPanel = ({ resources }) => {
+  const options = {
+    renderNode: {
+      [BLOCKS.EMBEDDED_ASSET]: node => {
+        let src, title
+
+        if (
+          node.data &&
+          node.data.target &&
+          node.data.target.fields &&
+          node.data.target.fields.file
+        ) {
+          src = node.data.target.fields.file["en-US"]
+            ? node.data.target.fields.file["en-US"].url
+            : null
+          title = node.data.target.fields.title
+            ? node.data.target.fields.title["en-US"]
+            : null
+        }
+        if (src)
+          return (
+            <img
+              className="resource__rich-img"
+              src={src}
+              alt={`${title ? title : ""}`}
+            />
+          )
+      },
+      [BLOCKS.PARAGRAPH]: (node, children) => (
+        <p className="resource__rich-text">{children}</p>
+      ),
+      [BLOCKS.LIST_ITEM]: (node, children) => (
+        <li className="resource-rich-li">{children}</li>
+      ),
+      [BLOCKS.QUOTE]: (node, children) => (
+        <blockquote className="resource__rich-quote">{children}</blockquote>
+      ),
+      // [INLINES.ENTRY_HYPERLINK]: (node, children) => (
+      //   // will need to build out this logic further
+      //   <a href={`/pages/temp`}>{children}</a>
+      // ),
+    },
+  }
+
+  const renderDescription = (descBody, excerptBody) => {
```

Comment:
I'm not sure this is working correctly. On the results page, I am seeing the description content and not the excerpt? It should only be the excerpt there, and only the description on the single page.

Reply:
You are right I had that backward. Just got that touched up. I will go ahead and set that spacing to all heading tags as well!